### PR TITLE
More storage for more things

### PR DESCRIPTION
**Pull request by spessbro**
## About The Pull Request
Lets the medical belt hold all masks and glass subtypes, lets biobags hold food except growns and lets construction bags hold material sheets

## Why It's Good For The Game
The glass subtype is used for proper reagent containers so it would make sense for the medical belt to be able to hold both beakers and drinking glasses and stuff of the like. As for masks there are a lot of them and limiting the types makes sense for a medical belt.
For the bio bag i always felt it was weird it couldn't hold food since you know, bio is in its name. When it comes to the construction bag this makes it more a poor man's rcd.

## Changelog
:cl:
tweak: medical belt can now hold all glass and mask subtypes
tweak:bio bags can hold food
tweak:construction bags can hold material sheets
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/belt.dm [hunk 1]
```diff
@@ -142,8 +142,7 @@
 		/obj/item/healthanalyzer,
 		/obj/item/dnainjector,
 		/obj/item/reagent_containers/dropper,
-		/obj/item/reagent_containers/glass/beaker,
-		/obj/item/reagent_containers/glass/bottle,
+		/obj/item/reagent_containers/glass/,
```
Comment (Fikou) [inline code/game/objects/items/storage/belt.dm:145]: everywhere in this file you have examples on how to do it right, and you still fuck it up

**Comment by XDTM** [comment]
https://github.com/tgstation/tgstation/blob/31dc0186ca3bd0e20793373c8ae196cc9f681d42/code/modules/reagents/reagent_containers/glass.dm#L232

File: code/game/objects/items/storage/belt.dm (hunk 1 above)
Comment (spessbro) [reply code/game/objects/items/storage/belt.dm:145]: hey i never said i was smark okay

**Comment by spessbro** [comment]
> https://github.com/tgstation/tgstation/blob/31dc0186ca3bd0e20793373c8ae196cc9f681d42/code/modules/reagents/reagent_containers/glass.dm#L232

yes this is the point

**Comment by XDTM** [comment]
https://github.com/tgstation/tgstation/blob/ffbcc8790dc63513eea9321f347b4fb119271161/code/modules/detectivework/footprints_and_rag.dm#L6

File: code/game/objects/items/storage/belt.dm (hunk 1 above)
Comment (ShizCalev) [inline code/game/objects/items/storage/belt.dm:145]:
```suggestion
		/obj/item/reagent_containers/glass,
```

File: code/game/objects/items/storage/belt.dm [hunk 2]
```diff
@@ -159,9 +158,7 @@
 		/obj/item/clothing/gloves/,
 		/obj/item/lazarus_injector,
 		/obj/item/bikehorn/rubberducky,
-		/obj/item/clothing/mask/surgical,
-		/obj/item/clothing/mask/breath,
-		/obj/item/clothing/mask/breath/medical,
+		/obj/item/clothing/mask/,
```
Comment (ShizCalev) [inline code/game/objects/items/storage/belt.dm:161]:
```suggestion
		/obj/item/clothing/mask,
```

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by ShizCalev** [comment]
keep in mind that this will allow aliens to hide facehuggers in the belt.

**Comment by Farquaar** [comment]
You can't just have an open-top beaker attached to your belt. That's what bottles are for. They come with nice little caps, at the cost of smaller capacity.

**Comment by gryffinp** [comment]
You can _already_ have an open-topped beaker attached to your belt.

**Comment by LemonInTheDark** [comment]
`For the bio bag i always felt it was weird it couldn't hold food since you know, bio is in its name` This feels a bit nonsensical, it's a bag used for viro isn't it?

**Comment by interestingusernam3** [comment]
>For the bio bag i always felt it was weird it couldn't hold food since you know, bio is in its name This feels a bit nonsensical, it's a bag used for viro isn't it?

And xeno and cyto

**Comment by LemonInTheDark** [comment]
I don't like this. Like I said above, some of the changes appear to be a bit nonsensical, and you're evening out storage based off type, which I think isn't the right way to go about it. Players don't think in terms of object type, and we shouldn't encourage them to.

I'm less against some of the other changes you've made here, if you're interested in pursuing it further poke me on discord and we can discuss it.

**Pull request closed without merging**